kolibri/core/webpack/management/commands/webpack_json.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+
+from kolibri.core.webpack.hooks import WebpackBundleHook
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    args = ()
+    help = 'Creates a new schema'  # @ReservedAssignment
+    option_list = BaseCommand.option_list + ()
+
+    def handle(self, *args, **options):
+
+        logging.debug(args)
+
+        for hook in WebpackBundleHook().registered_hooks:
```

Comment:
Yes, everything that's non-abstract and inherits from the WebpackBundleHook will be listed. So that's the basic common denominator for JS assets is that they are supposed to feed into webpack building I guess. Will be pretty cool to have everything properly managed like that.